package.json

```diff
@@ -37,10 +37,13 @@
   "dependencies": {
     "aws-sdk": "^2.337.0",
     "chai": "^4.2.0",
+    "fs": "0.0.1-security",
     "jasmine": "^3.1.0",
     "jasmine-promise": "^0.0.1",
+    "jasmine-reporters": "^2.3.2",
     "jasmine-ts-console-reporter": "^3.1.1",
     "lodash.ismatch": "^4.4.0",
+    "moment": "^2.22.2",
```

Comment:
consider date-fns instead of moment. It's lighter.

https://date-fns.org/

Reply:
done